### ISSUE TITLE
Local instance returns different results compared the public API

### ISSUE THREAD

**Issue by andrec93p** [komoot/photon#489]
Hello, we've been using the Photon public API for autocomplete and reverse geocoding while developing one of our projects, and we then decided to install a local instance.

We configured the Nominatim database on our server, imported an extract for the Italian regions we need from Geofabrik (https://download.geofabrik.de/europe/italy/centro-200909.osm.pbf) and configured the API and the updates. Our instance works great, but with one problem: the results returned by it are substantially different than the ones returned by the public API for the same queries.

When I say different, I don't mean that some results are missing or are in a different position, I can understand this is going to happen given the different dataset. I mean that the same results, referring to the exact same places and OSM entities, have a different content, and we can't understand why.

Here's the link to our instance, I'll leave it accessible for a while so you can try it.

http://95.217.75.221:2322/

To explain this weird behaviour, I'll take a city, Livorno, as an example. Many other entities have the same issues.

For the simple query with no additional parameter (http://photon.komoot.de/api/?q=livorno), this is one of the items.

```
{
   "geometry":{
      "coordinates":[
         10.3091456,
         43.5507411
      ],
      "type":"Point"
   },
   "type":"Feature",
   "properties":{
      "osm_id":42481,
      "osm_type":"R",
      "extent":[
         9.8881379,
         43.5954875,
         10.4195121,
         43.4197864
      ],
      "country":"Italy",
      "osm_key":"place",
      "osm_value":"city",
      "name":"Livorno",
      "state":"Tuscany"
   }
}
```

While in the results from our instance (http://95.217.75.221:2322/api/?q=livorno), I see this.

```
{
   "geometry":{
      "coordinates":[
         10.3091456,
         43.5507411
      ],
      "type":"Point"
   },
   "type":"Feature",
   "properties":{
      "osm_id":42481,
      "osm_type":"R",
      "extent":[
         9.8881379,
         43.5954875,
         10.4195121,
         43.4197864
      ],
      "country":"Italy",
      "osm_key":"boundary",
      "osm_value":"administrative",
      "name":"Livorno",
      "type":"city"
   }
}
```

As you can see, this item clearly refers to the same OSM object (they both have the ID 42481), but the one from the public API is marked as a city while the one from our instance is marked as a place with a _type_ field among the OSM data. The _state_ field is missing as well for some reason.

The weird thing is that, when you actually look at the OSM data for the entity (https://www.openstreetmap.org/relation/42481), our result is correct: the OSM entity 42481 Livorno is indeed an administrative boundary, not a place.

This difference may not seem concerning, but it's actually quite a problem for us, since our client performs different actions and manipulation on the data according the OSM values, so we're forced to change the behaviour despite using the same kind of API.

Before you ask:
- We did import the Wikipedia importance data, as suggested by Nominatim (https://nominatim.org/release-docs/develop/admin/Import/#downloading-additional-data), with no difference aside for the order of the items.
- We tried using the ready-to-use link for the whole planet as well (https://github.com/komoot/photon#installation), and the same behaviour still occurs.

Aside for our specific problem, I wonder: why is this happening? Why are the public API data different than the OSM ones? Are additional data used, or is some manipulation performed? If so, what? Can we find some kind of documentation about it? Is there something we're missing?

Thank you.

**Comment by lonvia** [comment]
The simple difference here is that you run the latest version of photon while photon.komoot.de is still on 0.3.2.

The missing state was a regression that has already been fixed with #483. The downloadable planet should already have the state fields again.

As for osm_key/osm_value, that looks like a regression, too. It used to return the key/value of the linked place node. I admit that I'm a bit surprised by this. However, if this was the previous behaviour it obviously needs to be changed back. We did not mean to introduce any API-incompatible changes with 0.3.3.
